roles/sap_ha_install_anydb_ibmdb2/tasks/platform/db2cm_cloud_msazure_vm.yml

```diff
@@ -58,15 +76,19 @@
   failed_when: not __sap_ha_install_anydb_ibmdb2_config_msazure_1.rc == 0 and not 'ip address is pingable' in __sap_ha_install_anydb_ibmdb2_config_msazure_1.stdout
 
 # Setting to be performed on both Primary and Secondary node
-- name: SAP HA AnyDB - IBM Db2 HADR (MS Azure) - Primary and Secondary Node - Provide MS Azure CLI credentials
+# Reasons for noqa:
+# - no-changed-when: Db2 mandates use of AWS CLI for credentials instead of credential configuration directly via IBM Db2 selected Pacemaker Shell (crmsh)
+- name: SAP HA AnyDB - IBM Db2 HADR (MS Azure) - Primary and Secondary Node - Provide MS Azure CLI credentials # noqa no-changed-when
   register: __sap_ha_install_anydb_ibmdb2_config_msazure_credentials
   when:
     - __sap_ha_install_anydb_ibmdb2_platform == "cloud_msazure_vm"
   ansible.builtin.shell: |
     az login --service-principal -u "{{ sap_ha_install_anydb_ibmdb2_msazure_app_client_id }}" -p "{{ sap_ha_install_anydb_ibmdb2_msazure_app_client_secret }}" --tenant "{{ sap_ha_install_anydb_ibmdb2_msazure_tenant_id }}"
 
 # db2cm script must run as root
-- name: SAP HA AnyDB - IBM Db2 HADR (MS Azure) - Primary Node - Create Linux Pacemaker Fence Agent instantiation using db2cm
+# Reasons for noqa:
+# - no-changed-when: Db2 command will error if already exists and not perform changes
+- name: SAP HA AnyDB - IBM Db2 HADR (MS Azure) - Primary Node - Create Linux Pacemaker Fence Agent instantiation using db2cm # noqa no-changed-when
   register: __sap_ha_install_anydb_ibmdb2_config_msazure_2
```

Comment:
Same here and in the following tasks: Place the Ansible module before registering its output, or provide a justification for this order.